### PR TITLE
Ahodges/RESpolls

### PR DESCRIPTION
**Pull request by lamagnifica**
@sarina @smarnach @Kelketek @mhoeber FYI here's the draft documentation for researchers describing the analytics events. @stroilova @shnayder I kept this pretty lean, let me know if you have any additions. @catong @srpearce this feature contributed by McKinsey will be in next week's release! Additional info can be found in the is PR: https://github.com/mckinseyacademy/xblock-poll/pull/13 and this read me: https://github.com/mckinseyacademy/xblock-poll#analytics
DOC-2086

### CODE REVIEW

**Comment by smarnach** [comment]
@lamagnifica I read through the event documentation – looks accurate to me.

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -3532,6 +3532,96 @@ the response.
...
+.. list-table::
+   :widths: 15 15 60
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Details
+   * - ``url_name``
+     - string
+     - The unique location identifier for the poll Xblock.
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Nit: XBlock

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -3532,6 +3532,96 @@ the response.
...
+.. list-table::
+   :widths: 15 15 60
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Details
+   * - ``url_name``
+     - string
+     - The unique location identifier for the survey Xblock.
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: XBlock

**Comment by srpearce** [comment]
:+1: LGTM. **LOVE** the new alphabet headings in the event_list file!

**Pull request merged**